test/val/val_memory_test.cpp

```diff
@@ -1975,6 +1972,35 @@ OpFunctionEnd
           "OpTypeRuntimeArray %_runtimearr_uint\n"));
 }
 
+TEST_F(ValidateMemory, VulkanUniformStructInsideRTAWithRuntimeDescriptorArrayGood) {
+  std::string spirv = R"(
+OpCapability RuntimeDescriptorArrayEXT
+OpCapability Shader
+OpExtension "SPV_EXT_descriptor_indexing"
+OpMemoryModel Logical GLSL450
+OpEntryPoint Fragment %func "func"
+OpExecutionMode %func OriginUpperLeft
+OpDecorate %array_t ArrayStride 4
+OpMemberDecorate %struct_t 0 Offset 0
+OpDecorate %struct_t BufferBlock
```

Comment:
This makes the uniform a SSBO. This decoration should be Block for a UBO.

Reply:
Done